src/main/java/ita/softserve/course_evaluation/controller/UserController.java

```diff
@@ -39,12 +39,12 @@ public UserController(UserService userService) {
 
 	@GetMapping("/{id}")
 	public ResponseEntity<UserDto> readById(@PathVariable(value="id") long id){
-		return new ResponseEntity<>(userService.readById(id),  HttpStatus.OK);
+		return new ResponseEntity                                                                                                                                                                                   <>(userService.readById(id),  HttpStatus.OK);
```

Comment:
Empty ResponseEntity?

Reply:
one part of code has been shifed to the right, corrected